source/_components/media_player.kef.markdown

```diff
@@ -0,0 +1,66 @@
+---
+layout: page
+title: "KEF Wireless speakers"
+description: "How to set up the KEF media player platform"
+date: 2018-11-28
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: kef.png
+ha_category: Media Player
+ha_release: 0.85
+---
```

Comment:
`ha_iot_class` is missing in this list.

Reply:
Thanks! 👍 

I'll have to update it a somewhat because of changes requested on the actual component. I haven't had time to do that due to Christmas. 

I'll reduce the logo size while I'm at it 😃 

Also, the ha_iot_class should be "Local Polling"